src/super_gradients/training/transforms/transforms.py

```diff
@@ -1277,21 +1318,40 @@ def _filter_box_candidates(box1, box2, wh_thr=2, ar_thr=20, area_thr=0.1):
     return (w2 > wh_thr) & (h2 > wh_thr) & (w2 * h2 / (w1 * h1 + 1e-16) > area_thr) & (ar < ar_thr)  # candidates
 
 
-def _mirror(image, boxes, prob=0.5):
+def _flip_horizontal_image(image: np.array):
```

Comment:
A correct type hint for numpy array is `np.ndarray`, not `np.array`. `np.array` is a method.